Libraries/Components/TabBarIOS/TabBarIOS.ios.js

```diff
@@ -84,7 +88,8 @@ class TabBarIOS extends React.Component<ViewProps & {
         barTintColor={this.props.barTintColor}
         barStyle={this.props.barStyle}
         itemPositioning={this.props.itemPositioning}
-        translucent={this.props.translucent !== false}>
+        translucent={this.props.translucent !== false}
+        hidden={this.props.hidden !== false}>
```

Comment:
you should use defaultProps for this

Reply:
Good catch! I incorrectly copied from the line above, but that isn't right. I didn't use `defaultProps` because it's not used anywhere else in the component—just wanted to be consistent. I've updated to fix the bug